deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: cloud-resource-operator
       containers:
         - name: cloud-resource-operator
-          image: quay.io/integreatly/cloud-resource-operator:v0.7.1
+          image: quay.io/integreatly/cloud-resource-operator:0.7.2
```

Comment:
This should also be `quay.io/integreatly/cloud-resource-operator:v0.7.2`

This is something that they want for the intly operator.

Reply:
Sorry thats added now